layouts/shortcodes/dbm-mysql-agent-config-examples.md

```diff
@@ -52,19 +64,19 @@ instances:
     custom_queries:
     - query: SELECT age, salary, hours_worked, name FROM hr.employees;
       columns:
-        - name: age
+        - name: mysql.custom.employee_age
           type: gauge
-        - name: salary
+        - name: mysql.custom.employee_salary
            type: gauge
-        - name: hours
+        - name: mysql.custom.employee_hours
```

Comment:
For MySQL & SQL Server custom queries, we append to the prefix `<INTEGRATION>.` automatically so we'll probably want something like `custom.employee_age` or just `employee_age`.